html/mycakeapp/src/Controller/AuctionController.php

```diff
@@ -219,4 +235,186 @@ public function home2()
 		])->toArray();
 		$this->set(compact('biditems'));
 	}
+
+	//list.ctpでの落札者か確認
+	public function checkBidderId($id)
+	{
+		$n = $this->Bidinfo->find()->where(['user_id' => $id])->count();
+		return $n > 0 ? true : false;
```

Comment:
```suggestion
		return ($n > 0);
```
今回は修正不要ですが、現場によっては冗長だと指摘される可能性があります